anet.yml

```diff
@@ -307,11 +307,13 @@ dictionary:
       homeView:
         location: [34.52, 69.16]
         zoomLevel: 10
+      leafletOptions:
+        attributionControl: false
     baseLayers:
       - name: OSM
         default: true
         type: tile
-        url: "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+        url: "http://{s}.tiles.wmflabs.org/bw-mapnik/{z}/{x}/{y}.png"
```

Comment:
To respond to my own comment: when trying to get https://b.tiles.wmflabs.org/bw-mapnik/15/11586/11449.png it turns out the certificate is invalid.
[The certificate is for `*.wmflabs.org`, which does *not* match `*.tiles.wmflabs.org`. So it appears that, at least until they fix their certificate, we need `http` after all…

Reply:
Well, we could just use `http://tiles.wmflabs.org/bw-mapnik/{z}/{x}/{y}.png` since any `{s}.tiles.wmflabs.org` points to the exact same IP address as `tiles.wmflabs.org`. So https it is!

Reply:
good find :+1: ! pushed